x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/FlushJobAction.java

```diff
@@ -88,6 +92,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeOptionalString(advanceTime);
             out.writeOptionalString(skipTime);
             out.writeBoolean(waitForNormalization);
+            out.writeBoolean(refreshRequired);
```

Comment:
This needs BWC protection. Please read the JavaDocs in `TransportVersion.java` to see how to do it now.